src/components/Blog/Feed/Item/index.tsx

```diff
@@ -103,11 +107,13 @@ export const query = graphql`
     description
     descriptionLong
     picture {
-      big: gatsbyImageData(
-        width: 650
-        height: 450
-        transformOptions: { cropFocus: CENTER }
-      )
+      childImageSharp {
+        big: gatsbyImageData(
+          width: 650
+          height: 450
+          transformOptions: { cropFocus: CENTER }
+        )
+      }
     }
```

Comment:
These `picture` GraphQL endpoints now point to `File` nodes instead of `ImageSharp` nodes, using a simpler query method under the hood as well as paving the way for supporting files `sharp` doesn't support, like `SVG`s.